subchannel_test.go

```diff
@@ -195,8 +195,7 @@ func TestGetHandlers(t *testing.T) {
 	}{
 		{
 			serviceName: ch.ServiceName(),
-			// Default service name comes with extra introspection methods.
-			wantMethods: []string{"_gometa_introspect", "_gometa_runtime", "method1", "method2"},
+			wantMethods: []string{"method1", "method2"},
```

Comment:
To confirm this removal: before we registered both introspection and actual procedure together. Now we don't and those introspection are not visible anymore during actual handling.

Reply:
Yep exactly. They're no longer just "user-level" methods, but internal methods that are always registered at a higher layer, so they don't show up here anymore.